### ISSUE TITLE
Borked CSS Revision ccc0ded

### ISSUE THREAD

**Issue by rudijs** [kwk/docker-registry-frontend#63]
Hi,

I'm using revision ccc0ded and the CSS is not loading for me.

Here's a screencap to help.

It does show the CSS file being downloaded in the browser OK (status 200) - just not rendering.

![selection_014](https://cloud.githubusercontent.com/assets/1711940/10021386/4fc3a8ae-617a-11e5-8117-17602c2ac8e5.jpg)

**Comment by mjgsilva** [comment]
+1.

@kwk Any idea why is this happening?

**Comment by dsmorse** [comment]
Yes, I'm seeing the same thing, the ccc0ded has the fix from 53, but I still do not see styles... Is there an older tag I can pull , that anyone knows was working?

**Comment by kwk** [comment]
Here's an idea. Have you wiped your browsers cache? Whenever I made changes to the UI my colleagues wondered why they couldn't see them and it was mostly due to an old cached site. Just an idea...

**Comment by dsmorse** [comment]
I experienced it on a fresh install on a new URL... so I doubt cache played a role, but I can try.  I pulled the V1-deprecrated tag and that one works with the style sheets.

**Comment by mjgsilva** [comment]
^ Same here, I did a fresh installment.
Tested with Chrome, Firefox and Safari - same result, css not rendering.

**Comment by FredrikWendt** [comment]
Me too (and everyone else at the job that I've spoken to today).

**Comment by kwk** [comment]
I'm sorry for this bad experience. Can anybody please try out running the `v1-deprecated` tag? The instructions to run it can be found [here](https://github.com/kwk/docker-registry-frontend/tree/v1-deprecated). @dsmorse seem to have it working with `v1-deprecated`. Thank you in advance.

**Comment by mvdkleijn** [comment]
v1-deprecated seems to work for me
